pkg/operator/staticpod/controller/backingresource/backing_resource_controller.go

```diff
@@ -0,0 +1,197 @@
+}
+
+func NewBackingResourceController(
+	targetNamespace string,
+	operatorConfigClient common.OperatorClient,
+	kubeInformersForTargetNamespace informers.SharedInformerFactory,
+	kubeClient kubernetes.Interface,
+) *BackingResourceController {
+	c := &BackingResourceController{
+		targetNamespace:      targetNamespace,
+		operatorConfigClient: operatorConfigClient,
+
+		saListerSynced: kubeInformersForTargetNamespace.Core().V1().ServiceAccounts().Informer().HasSynced,
+		saLister:       kubeInformersForTargetNamespace.Core().V1().ServiceAccounts().Lister(),
+
+		clusterRoleBindingListerSynced: kubeInformersForTargetNamespace.Core().V1().ServiceAccounts().Informer().HasSynced,
+		clusterRoleBindingLister:       kubeInformersForTargetNamespace.Rbac().V1().ClusterRoleBindings().Lister(),
+
+		queue:      workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "BackingResourceController"),
+		kubeClient: kubeClient,
+	}
+
+	operatorConfigClient.Informer().AddEventHandler(c.eventHandler())
+
+	kubeInformersForTargetNamespace.Core().V1().ServiceAccounts().Informer().AddEventHandler(c.eventHandler())
+	kubeInformersForTargetNamespace.Rbac().V1().ClusterRoleBindings().Informer().AddEventHandler(c.eventHandler())
+
+	return c
+}
+
+// resetFailingConditionForReason reset the failing operator condition status to false for the specified reason.
+func (c BackingResourceController) resetFailingConditionForReason(status *operatorv1alpha1.StaticPodOperatorStatus, resourceVersion, reason string) error {
+	failingCondition := v1alpha1helpers.FindOperatorCondition(status.Conditions, operatorv1alpha1.OperatorStatusTypeFailing)
+	if failingCondition == nil || failingCondition.Reason != reason {
+		return nil
+	}
+
+	failingCondition.Status = operatorv1alpha1.ConditionFalse
+	v1alpha1helpers.SetOperatorCondition(&status.Conditions, *failingCondition)
+
+	_, err := c.operatorConfigClient.UpdateStatus(resourceVersion, status)
+	return err
+}
+
+func (c BackingResourceController) templateAssetFn(name string) ([]byte, error) {
+	config := struct {
+		TargetNamespace string
+	}{
+		TargetNamespace: c.targetNamespace,
+	}
+	return assets.MustCreateAssetFromTemplate(name, bindata.MustAsset(filepath.Join(manifestDir, name)), config).Data, nil
+}
+
+func (c BackingResourceController) sync() error {
+	operatorSpec, originalOperatorStatus, resourceVersion, err := c.operatorConfigClient.Get()
+	if err != nil {
+		return err
+	}
+
+	operatorStatus := originalOperatorStatus.DeepCopy()
+	switch operatorSpec.ManagementState {
+	case operatorv1alpha1.Unmanaged:
+		return nil
+	case operatorv1alpha1.Removed:
+		// TODO: Should we delete the installer-sa and cluster role binding?
+		return nil
+	}
+
+	errors := []string{}
+	directResourceResults := resourceapply.ApplyDirectly(c.kubeClient, c.templateAssetFn,
+		"manifests/installer-sa.yaml",
+		"manifests/installer-cluster-rolebinding.yaml",
+	)
+
+	for _, currResult := range directResourceResults {
```

Comment:
I think we want general helper in `resourceapply` that does this. We're doing it in many spots.

Reply:
a follow up?

Reply:
> a follow up?

@sanchezl or @damemi perhaps?  This week would be good.